keyboards/1upkeyboards/pi40/keymaps/miketronic/keymap.c

```diff
@@ -0,0 +1,181 @@
+  * ├─────┼─────┼─────┼─────┼─────┼─────┤   ├─────┼─────┼─────┼─────┼─────┼─────┤
+  * │ FN  │     │     │     │     │     │   │     │     │     │     │     │     │
+  * │     │     │     │     │     │     │   │     │     │     │     │     │     │
+  * └─────┴─────┴─────┴─────┴─────┴─────┘   └─────┴─────┴─────┴─────┴─────┴─────┘
+  */
+  [_FN] = LAYOUT_ortho_4x12_wrapper (
+                                                                                                                            KC_MUTE,
+        TD_CP,     CLIP,      SS,        _______,   _______,   _______,   _______,     TD_SL,  _______,    KC_UP,  _______,  KC_DEL,
+        KC_TAB,    M05,       M06,       M07,       M08,       _______,   _______,   _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,   TD_UR,
+        _______,   M01,       M02,       M03,       M04,       _______,   _______,   _______,  TD_QEXC,  KC_COMM,   KC_DOT,   TD_DL,
+        _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,  _______
+  ),
+
+
+  /* SYMBOLS
+  * ┌─────┬─────┬─────┬─────┬─────┬─────┐   ┌─────┬─────┬─────┬─────┬─────┬─────┐
+  * │     |  $  |  ^  |  <  |  >  │ SYMB│   │ SYMB│  |  |  [  |  ]  |  -  |     |
+  * │     │     │     │     │     │     │   │     │     │     │     │     │     │
+  * ├─────┼─────┼─────┼─────╆━━━━━╅─────┤   ├─────╆━━━━━╅─────┼─────┼─────┼─────┤
+  * │     |  %  |  :  |  ;  ┃  +  ┃  =  │   │     ┃     ┃  (  |  )  |  _  |     |
+  * │     │     │     │     ┃     ┃     │   │     ┃     ┃     │     │     │     │
+  * ├─────┼─────┼─────┼─────╄━━━━━╃─────┤   ├─────╄━━━━━╃─────┼─────┼─────┼─────┤
+  * |     |  &  |  '  |  "  |  *  |  #  │   │  !  |  ?  |  /  |  \  |  @  |     | 
+  * │     │     │     │     │     │     │   │     │     │     │     │     │     │
+  * ├─────┼─────┼─────┼─────┼─────┼─────┤   ├─────┼─────┼─────┼─────┼─────┼─────┤
+  * │ FN  │     │     │     │     │     │   │     │     │     │     │     │     │
+  * │     │     │     │     │     │     │   │     │     │     │     │     │     │
+  * └─────┴─────┴─────┴─────┴─────┴─────┘   └─────┴─────┴─────┴─────┴─────┴─────┘
+  */
+
+  [_SYMB] = LAYOUT_ortho_4x12_wrapper (
+                                                                                                                       KC_MUTE,
+        _______,  KC_DLR,   KC_CIRC,  KC_LABK,  KC_RABK,  _______,  _______, KC_PIPE,   KC_LBRC,   KC_RBRC,  KC_MINS,  _______,
+        _______,  KC_PERC,  KC_COLN,  KC_SCLN,  KC_PLUS,  KC_EQL,   _______, _______,   KC_LPRN,   KC_RPRN,  KC_UNDS,  _______,
+        _______,  KC_AMPR,  KC_QUOT,  KC_DQUO,  KC_ASTR,  KC_HASH,  KC_EXLM,   KC_QUES,   KC_SLSH,  KC_BSLS,   KC_AT,  _______,
+        _______,  WM,       _______,  _______,  _______,  _______,  _______, _______,   _______,   _______,  _______,  _______
+  )
+
+};
+
+#if defined(ENCODER_MAP_ENABLE) && defined(KEYBOARD_1upkeyboards_pi40_mit_v1_0)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
```

Comment:
Just a heads up, this has been changed on develop: 
```c
const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
```

Reply:
If I change it now will it cause errors since it's in develop?

Reply:
Yeah.  You can target develop, and that would allow it to work properly

Also, if you wanted, you could add this before the encoder_map array, and use the above code: 
```c
#ifndef NUM_DIRECTIONS
#    define NUM_DIRECTIONS 2
#endif
```